packages/form-layout/test/visual/lumo/form-layout.test.js

```diff
@@ -101,6 +101,39 @@ describe('form-layout', () => {
     it('colspan', async () => {
       await visualDiff(element, 'colspan');
     });
+
+    it('colspan alignment', async () => {
+      element = fixtureSync(`
```

Comment:
Let's extract this to a separate suite if it requires a different fixture.

Reply:
Done.